### DISCUSSION TITLE
[FR] Support for multiple wireguard configs

### DISCUSSION THREAD

**Discussion by DroidFreak32** [pivpn/pivpn#1501] [Feature requests]
**Is your feature request related to a problem? Please describe.**
I set up 2 Wireguard Interfaces wg0 and wg1337

- `wg0` is the generic PiVPN interface that uses my PiHole for DNS filtering - No issues here.
- `wg1337` is kind of like a bridge to a VPN interface which I use for downloading Linux torrents every day.

All clients connected to `wg0` (172.16.1.0/24) are just used for accessing my home network + Pihole
All clients connected to `wg1337` (172.16.2.0/24) get their traffic forwarded to a VPN interface

I used to manually generate configs for both before I discovered PiVPN
With PiVPN it makes my life easy to create wg0 and manage clients on 172.16.1.0/24
However adding clients on wg1337 is still a manual procedure and cannot be tracked easily.

**Describe the solution you'd like**
Add another paramenter such as `--config` to support different SetupVars.conf
Bonus: Associate clients and its SetupVars.conf. `/etc/pivpn/wireguard/`can be used to store setups for multiple wireguard interfaces

**Describe alternatives you've considered**
Creating a manual cronjob to copy wg0.conf to wg1773.conf by changing the IP address subnet.

**Comment by coolapso** [comment]
Moving to Discussions.

**Comment by DroidFreak32** [comment]
May have to wait for PR 1467 to be ready first. my POC PR will have conflicts and 1467 has a lot of important additions, along with ipv6 support.

**Reply by orazioedoardo** [reply]
Pull request #1467 should work, but needs testing.
